src/resources/startup.py

```diff
@@ -319,5 +358,11 @@ def sigterm_handler(signum, frame):
 
     server.add_insecure_port(FLAGS.socket)
     server.start()
+    time.sleep(4)
```

Comment:
Why wait here on the main thread. Can't you just change the location of the sleep in the background thread?

Reply:
Sure, I will update it and remove it from here.